sources/packages/web/src/components/layouts/aest/AESTApplicationSideBar.vue

```diff
@@ -41,6 +41,27 @@
       :title="studentMenu.applicationStatus.title"
       @click="studentMenu.applicationStatus.command"
     />
+    <v-list v-if="applicationHistory.length" density="compact" nav>
+      <v-list-group
+        v-for="application in applicationHistory"
+        :key="application.title"
+        :prepend-icon="application.props?.prependIcon"
+        :title="application.title"
+        :value="application.title"
+      >
+        <template v-slot:activator="{ props }">
+          <v-list-item v-bind="props" :title="application.title"></v-list-item>
+        </template>
+
+        <v-list-item
+          v-for="child in application.children"
+          :key="child.title"
+          :title="child.title"
+          :prepend-icon="child.props?.prependIcon"
+          @click="child.command"
```

Comment:
As per my share in DEV chat, please refrain from using the `@click` and use `to:` rather.